source/fundamentals/builders/indexes.txt

```diff
@@ -11,3 +11,181 @@ Indexes Builders
    :class: singlecol
 
 .. _indexes-builders:
+
+Overview 
+--------
+
+This guide shows you how to specify **indexes** using the Java Driver
+:doc:`builders </fundamentals/builders/>`. Index builders provide
+helper methods for the following types of indexes:
+
+- :ref:`Ascending <index-ascending>`
+- :ref:`Descending <index-descending>`
+- :ref:`Compound Indexes <index-compound-indexes>`
+- :ref:`Text Indexes <index-text-indexes>`
+- :ref:`Hashed Indexes <index-hashed-indexes>`
+- :ref:`Geospatial Indexes <index-geospatial-indexes>`
+
+Indexes store a subset of the collection’s data set. The index stores
+the value of a specific field or set of fields, ordered by the value of
+the field. See our guide on :doc:`Indexes </fundamentals/indexes>` to
+see examples of queries covered by indexes. 
```

Comment:
Nitpick: sorry, missed this in my suggestion from the previous review. No change necessary, but it might be better to ask them to "see" only once.
```suggestion
the field. See our guide on :doc:`Indexes </fundamentals/indexes>` for
examples of queries covered by indexes. 
```